include/metrics.hrl

```diff
@@ -47,6 +48,8 @@
     {?METRICS_VM_CPU, prometheus_gauge, [cpu], "Router CPU usage"},
     {?METRICS_VM_PROC_Q, prometheus_gauge, [name], "Router process queue"},
     {?METRICS_VM_ETS_MEMORY, prometheus_gauge, [name], "Router ets memory"},
-    {?METRICS_XOR_FILTER, prometheus_gauge, [], "Router XOR Filter udpates"},
+    {?METRICS_XOR_FILTER, prometheus_counter, [], "Router XOR Filter udpates"},
```

Comment:
This is gauge because the metric is the cost in DC of an XOR txn.

Reply:
Easier to read I think if a counter where it only increment with a new submit, no?